saturn-rapids/environment.yml

```diff
@@ -0,0 +1,31 @@
+name: saturn
+channels:
+- nvidia
+- rapidsai
+- numba
+- defaults
+- conda-forge
+dependencies:
+- blas=*=mkl
+- bokeh
+- cvxpy
+- dask-ml
+- dask=2.30.0
+- distributed=2.30.1
+- ipykernel
+- ipywidgets
+- matplotlib
+- numpy
+- pandas
+- pip
+- prefect
+- pyarrow
+- python=3.7
+- rapids=0.17.0=cuda10.1_py37_g180f433_165
+- s3fs
+- voila
```

Comment:
For consistent with the saturn image, I think this one should have `scipy`. With rapids people tend to have to do some things in cpu sometimes, so I would like it to have parity with the saturn image, except with the added rapids/gpu stuff